### DISCUSSION TITLE
tree-sitter don't work on wsl.

### DISCUSSION THREAD

**Discussion by zhang-yunqi** [helix-editor/helix#9579] [Troubleshooting]
### Summary

Hello, my helix can't hightlight in any language even toml config file, but neovim works well. I installed it with pacman. how can I slove the question.
![this is healthe](https://github.com/helix-editor/helix/assets/82689027/3ab5439c-416d-4db2-887d-c4c87fa7abfc)
 

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:
Highlight page like this
![image](https://github.com/helix-editor/helix/assets/82689027/7823c052-d96b-4308-9246-a0416c5169ee)

Instead, this happened:
tree-sitter didn't work like this
![image](https://github.com/helix-editor/helix/assets/82689027/e473e2af-3b6b-49e9-ac93-9d8bb4d1f7f1)

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Arch Linux(wsl)

### Terminal Emulator

windows terminal

### Installation Method

pacman

### Helix Version

helix-git

**Comment by archseer** [comment]
Looks like your runtime is configured incorrectly because all the highlight columns are marked with a red X.
